src/ir/effects.h

```diff
@@ -470,7 +484,12 @@ struct EffectAnalyzer
   }
   void visitNop(Nop* curr) {}
   void visitUnreachable(Unreachable* curr) { branchesOut = true; }
-  void visitPop(Pop* curr) { calls = true; }
+  void visitPop(Pop* curr) {
+    // When EH is enabled, any call can throw.
```

Comment:
`call` seems irrelevant here?

Reply:
Oh, thanks, deleted the comment. I was copy-pasting and didn't end up deleting that I guess. Or are you talking about that I deleted `calls = true;`? I'm not sure why `calls` was set to true for `pop`s in the first place, it was that way when `pop` support was first added by you. I guess you did that to prevent `pop`s from being moved willy-nilly when they should't be moved I guess. But now we have `danglingPop`, which sets `hasSideEffects()` to true, I think the same effect can be achieved by that.

Reply:
Yes, I meant the comment looked wrong.